### DISCUSSION TITLE
How to use older image version?

### DISCUSSION THREAD

**Discussion by rasyidstat** [microsoft/PlanetaryComputer#69] [Q&A] [answered]
Hi team,

How can I use the older image version? 

Since 21 May 2022, my code breaks. I am wondering whether the break happens because of the update on the base image [pangeo-docker-images](https://github.com/pangeo-data/pangeo-docker-images).

I got this error message on my code

```
Invalid projection: EPSG:None: (Internal Proj Error: proj_create: crs not found)
```

I use this Dockerfile

```
FROM mcr.microsoft.com/planetary-computer/python
```

Do you have any suggestions on which older image version I can use?

Thank you

**Comment by TomAugspurger** [comment, marked as answer]
Hi @rasyidstat. Do you have a minimal example that reproduces that error? It'd be good to report that to the upstream project if it is indeed a regression.

This images are built at https://github.com/microsoft/planetary-computer-containers. You can view the list of tags at https://mcr.microsoft.com/v2/planetary-computer/python/tags/list. The recent release moved from 2022.02.14.0 to 2022.05.11.0 (https://github.com/microsoft/planetary-computer-hub/pull/65/files)

If you're using the Hub, then unfortunately there isn't an easy way for you to pick a specific image. If you're using something like [Codespaces](https://planetarycomputer.microsoft.com/docs/overview/ui-codespaces/) or just [our Dask Gateway](https://planetarycomputer.microsoft.com/docs/concepts/computing/#use-our-dask-gateway) then you can specify an image. And if you deploy your own hub you can use any image you like.

**Reply by rasyidstat** [reply]
Hi @TomAugspurger 

Thank you for the explanation. 

Unfortunately, it is difficult to reproduce and debug the error at this moment. I use my own hub for the deployment.

Is it correct that the Dockerfile should be changed to this to use the older version?

```
FROM mcr.microsoft.com/planetary-computer/python:2022.02.14.0
```

**Reply by TomAugspurger** [reply]
Yep, was the environment used prior to a couple weeks ago.
